examples/flutter_gallery/lib/demo/cupertino/cupertino_text_field_demo.dart

```diff
@@ -167,25 +171,28 @@ class _CupertinoTextFieldDemoState extends State<CupertinoTextFieldDemo> {
           middle: Text('Text Fields'),
         ),
         child: CupertinoScrollbar(
-          child: ListView(
-            children: <Widget>[
-              Padding(
-                padding: const EdgeInsets.symmetric(vertical: 32.0, horizontal: 16.0),
-                child: Column(
-                  children: <Widget>[
-                    _buildNameField(),
-                    _buildEmailField(),
-                    _buildLocationField(),
-                    _buildPinField(),
-                    _buildTagsField(),
-                  ],
+          child: ExampleAutofillForm(
+            uniqueIdentifier: 'Form',
+            child: ListView(
+              children: <Widget>[
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: 32.0, horizontal: 16.0),
+                  child: Column(
+                    children: <Widget>[
+                      _buildNameField(),
+                      _buildEmailField(),
+                      _buildLocationField(),
+                      _buildPinField(),
+                      _buildTagsField(),
+                    ],
+                  ),
                 ),
```

Comment:
As discussed in the emails, if we implement autofill milestone per milestone, can we remove the need for a form in the first milestone?

Reply:
I think we still need some sort of scope for autofill, be it an existing widget (e.g., `Form`) or a new widget. Offstage input fields shouldn't be involved in autofill.

Reply:
If they are not built, they won't get invoked by platform autofill call.
If they are not on the screen (may be on the corner built but not visible to the user), I believe it is still ok to send the text field's value.

Reply:
I might be missing something though.. Let's discuss more in person

Reply:
One scenario is that the autofill service could overwrite some fields without the user knowing. It gets worse if the user couldn't easily check these fields after autofill (for instance, autofill filled something that's in the previous route).